src/renderer/features/governance/components/Delegations/TotalDelegation.tsx

```diff
@@ -66,14 +66,15 @@ export const TotalDelegation = ({ onClick }: Props) => {
             <FootnoteText>{t('governance.delegations.label')}</FootnoteText>
           </div>
 
-          {isLoading && <Shimmering width={120} height={20} />}
-          {!isLoading &&
-            asset &&
-            (totalDelegation !== '0' ? (
-              <AssetBalance className="text-small-title" value={totalDelegation} asset={asset} />
+          {!isLoading && network?.asset ? (
```

Comment:
You can check network against null and it will be enough
```suggestion
          {!isLoading && network ? (
```